twoplayer.s

```diff
@@ -34,7 +35,22 @@
 
 .segment "GAMEBSS"
```

Comment:
Then how about a segment to add variables to zeropage? Is it possible/helpful? I currently just picked some (hopefully) unused adresses manually.

Reply:
Take a look at `build/tetris-ram.s`. The unlabeled `.res`s are believed-to-be-unused (although for some of them there are important details, like `playfield` actually is a $100 long because some code doesn't handle over/underflow, but only $C8 contains the playfield on the screen, but not all of that $100 is read via over/underflow).

This GAMEBSS was easy because the game doesn't use the entire range of .bss; there's extra space _at the end_. So we just define some more contents in .bss and the linker puts things together without issue. Zeropage is harder because 1) it is fragmented, and 2) the freespace isn't at the end. I can "punch" holes, but each time requires two segments and choosing which segment your variables go in.

Up to this point, I've mainly been _reusing_ variables in ZP that I know are unused. There is a 24 byte space after `spawnCount`, so maybe it would be good to cut out a space there for reserving ZP variables.